lib/single-new-eks-fargate-opensource-observability-pattern/index.ts

```diff
@@ -59,6 +59,11 @@ export default class SingleNewEksFargateOpenSourceObservabilityConstruct {
         console.log(doc);
         fs.writeFileSync(__dirname + '/../common/resources/otel-collector-config-new.yml', doc);
 
+        if (utils.valueFromContext(scope, "adotcollectormetrics.pattern.enabled", false)) {
+            ampAddOnProps.openTelemetryCollector = {
```

Comment:
same feedback. doc config render is not consistent with OSS pattern. Except for Istio logic, everything else can be exactly copy pasted here.